environment.yml

```diff
@@ -49,8 +49,8 @@ dependencies:
   - conda-forge::rdkit >=2022.09.1
 
 # general-purpose external software tools
-  - conda-forge::julia=1.9.1
-  - conda-forge::pyjulia >=0.6
+  - conda-forge::juliaup
```

Comment:
This makes sense. What does this mean as far as how we think about julia version?

Reply:
Chiming in with some notes from offline conversation, just to make sure I'm on the same page as well. From my understanding, `juliaup` installs a Julia binary in the conda environment, but 'hides' it from conda so that that `juliaup` can manage the version. I think this results in the Julia version being whatever the default provided by `juliaup` is, which I think is just the latest minor release.

Reply:
We can also explicitly manage julia version through commands such as `juliaup add 1.9.0`. I incline to let the Julia version update whenever we could and avoid sticking to an old version.